ci-operator/step-registry/cucushift/upgrade/helper-service/build-info/cucushift-upgrade-helper-service-build-info-commands.sh

```diff
@@ -12,3 +12,10 @@ if [[ -n "${RELEASE_IMAGE_LATEST}" ]]; then
     LATEST_VERSION="$(oc adm release info "${RELEASE_IMAGE_LATEST}" --output=jsonpath="{.metadata.version}")"
     echo "${LATEST_VERSION}" > "${BUILD_INFO_FILE}"
 fi
+if [[ -n "${RELEASE_IMAGE_TARGET}" ]]; then 
+    echo "${RELEASE_IMAGE_TARGET}"
```

Comment:
Can we remove this line? The line following it already shows the info

Reply:
@jhou1 added this line for debug information and while testing.

Reply:
fixed in new commit.